src/__tests__/footer.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import Footer from '../Components/Footer/Footer.tsx';
+
+beforeAll(() => {
+  process.env = Object.assign(process.env, { NODE_ICU_DATA: 'node_modules/full-icu' });
+});
+
+global.fetch = jest.fn(() =>
+  Promise.resolve({
+    json: () => Promise.resolve({ version: '0.1.0'}),
```

Comment:
We should source version from `package.json` before this test. @guzman-raphael To check LabBook.